CONTRIBUTING.md

```diff
@@ -59,6 +59,22 @@ During the collector release process, all `./.chloggen/*.yaml` files are transcr
 
 Alternately, copy `./.chloggen/TEMPLATE.yaml`, or just create your file from scratch.
 
+## Platform Agnostic Code
+
+In order to ensure compatibility with different operating systems, code should be platform-agnostic. Below are some guidelines to follow when writing platform-agnostic code:
+
+* Avoid using platform-specific libraries, features etc. Please opt for cross-platform solutions. 
+
+* Avoid hard-coding platform specific values. Use environment variables or configuration files for storing platform specific values.
+
+* Be mindful of 
+  - Standard file systems and file paths such as forward slashes (/) instead of backward slashes (\) in Windows. 
```

Comment:
Can we provide specific guidance here such as to use the [`path/filepath` package](https://pkg.go.dev/path/filepath) when working with file paths?